mage_ai/frontend/components/ServerTimeDropdown/index.style.tsx

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+
+import dark from '@oracle/styles/themes/dark';
+
+const SHARED_CELL_PADDING = '12px 20px';
+const SHARED_GAP = '1px';
+
+export const DropdownContainerStyle = styled.div<{
+  top?: number;
+}>`
+  position: absolute;
+  top: ${props => props.top || 0};
+  right: 0;
+  width: 360px;
```

Comment:
If a unit value is a multiple of 8, we usually import `UNIT` from `@oracle/styles/units/spacing` and multiply that by a number to come up with the total value. For example, we might put `width: ${UNIT * 45}px;` here instead. The `UNIT` of 8 is basically the standard pixel unit Mage uses throughout the app.